test/app.test.ts

```diff
@@ -374,19 +393,32 @@ test('apps with varying yamlOutputTypes; chart dependencies via custom construct
   const testSpecs = [
     {
       props: { yamlOutputType: YamlOutputType.FILE_PER_CHART },
+      subFolders: false,
       result: ['0000-chart2.k8s.yaml', '0001-chart1.k8s.yaml'],
     },
     {
       props: { yamlOutputType: YamlOutputType.FILE_PER_APP },
+      subFolders: false,
       result: ['app.k8s.yaml'],
     },
     {
       props: { yamlOutputType: YamlOutputType.FILE_PER_RESOURCE },
+      subFolders: false,
       result: [
         'CustomConstruct.chart1-microservice-microserviceobj-c8e1164f.k8s.yaml',
         'CustomConstruct.chart2-database-databaseobj-c8b5eba3.k8s.yaml',
       ],
     },
+    {
+      props: { yamlOutputType: YamlOutputType.FOLDER_PER_CHART_FILE_PER_RESOURCE },
+      subFolders: true,
+      result: [
+        '0000-chart2',
+        'CustomConstruct.chart2-database-databaseobj-c8b5eba3.k8s.yaml',
+        '0001-chart1',
+        'CustomConstruct.chart1-microservice-microserviceobj-c8e1164f.k8s.yaml',
+      ],
```

Comment:
Please use a more generic helper method that lists all files in a directory (recursively), so that the output instead looks like:
```suggestion
      result: [
        '0000-chart2/CustomConstruct.chart2-database-databaseobj-c8b5eba3.k8s.yaml',
        '0001-chart1/CustomConstruct.chart1-microservice-microserviceobj-c8e1164f.k8s.yaml',
      ],
```
I would prefer this since it doesn't require making any assumptions about the structure of the output directory, and eliminates the extra test parameter `subFolders`.